frontend/packages/console-app/src/components/network-policies/network-policy-add-peer-dropdown.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Dropdown } from '@console/internal/components/utils';
+
+export type NetworkPolicyPeerType = 'sameNS' | 'anyNS' | 'ipBlock';
+
+export const NetworkPolicyAddPeerDropdown: React.FunctionComponent<NetworkPolicyAddPeerDropdownProps> = (
+  props,
+) => {
+  const { t } = useTranslation();
+  const { title, onSelect } = props;
+  const options = [
+    t('public~Allow pods from the same namespace'),
+    t('public~Allow pods from inside the cluster'),
+    t('public~Allow peers by IP block'),
+  ];
+  return (
+    <div className="form-group co-create-networkpolicy__add-peer">
+      <Dropdown
```

Comment:
Note: For new dropdowns, you should prefer the PatternFly dropdown. We're trying to move away from the internal one.

OK to leave as-is for this PR, though.